src/app/services/scheduler/scheduler.service.ts

```diff
@@ -350,7 +350,7 @@ export class SchedulerService {
             if (resource[`ephemeral-storage`] == 0) {
               formatted.push(`ephemeral-storage: ${NOT_AVAILABLE}`);
             }else{
-              formatted.push(`ephemeral-storage: ${CommonUtil.formatBytes(resource[key])}`);
+              formatted.push(`ephemeral-storage: ${CommonUtil.formatEphemeralStorageBytes(resource[key])}`);
```

Comment:
All `hugepages-*` resources fall into a special category also. They are memory sizes and should be displayed as such.
Can use the multiple case value support javascript has here if we use  `formatted.push(`${key}: ...`

Reply:
I added it in the refactored code. Thanks for catching that!